outputs/structured.go

```diff
@@ -75,6 +75,10 @@ func (r Structured) Output(w io.Writer, results <-chan []resource.TestResult, st
 
 	fmt.Fprintln(w, string(j))
 
+	if result.Summary.Failed > 0 {
```

Comment:
Hope you don't mind, I fixed what I think is a bug, here. I think all outputs (except the nagios one?) behave like this change has made this one behave here.

Reply:
(If this was intended behaviour, I can choose a different output to test against easily enough.)

Reply:
Having read the log of that file, I reckon this was by-design now. So, I've removed the change and aimed my test at the rspecish output format instead.